app/static/js/initialize_map.js

```diff
@@ -0,0 +1,119 @@
+geocoder = new google.maps.Geocoder();
+address = document.getElementById("address");
+
+// Global array for map markers
+var markers = []
+
+// Global Marker Wrappers and Map
+var globalMarkerWrappers = null;
+var globalMap = null;
+
+// Initial map center coordinates
+INITIAL_CENTER_LAT = 39.952;
```

Comment:
You should recenter using something like [this](http://stackoverflow.com/questions/19304574/center-set-zoom-of-map-to-cover-all-markers-visible-markers)

That way you won't need to hardcode a center